Jenkinsfile.github

```diff
@@ -16,8 +16,24 @@ pipeline {
         GIT_REPO_NAME = getRepoName()
         RELEASE_BRANCH_VERSION = getReleaseBranchVersion()
     }
-    
+
     stages {
+        stage("Update Vendored Dependencies") {
+            steps {
+                script {
+                    sh(returnStdout: true, script: "git vendor list").split("\n").findAll{it =~ /^[a-z]/}.each {
+                        it = it.replaceFirst(/:$/, "")
+                        echo "Updating vendor dependency ${it}"
+                        (vendorName, vendorBranch) = it.split('@')
+                        sh "git vendor update ${vendorName} ${vendorBranch}"
```

Comment:
In case of outdated dependency, "git vendor update" command actually creates 2 commits (one is a squash of remote repo, another is update of subtree reference).